lib/render.js

```diff
@@ -88,7 +88,13 @@ function renderComponent(req, res, hrStart) {
     .then(cmptData => formDataForRenderer(cmptData, { _ref }, locals))
     .tap(logTime(hrStart, _ref))
     .then(({data, options}) => renderer.render(data, options, res))
-    .catch(err => log('error', err));
+    .catch(err => {
+      const errorHandler = responses.handleError(res);
+
+      log('error', err);
+
+      return errorHandler(err);
```

Comment:
```suggestion
      return responses.handleError(res)(err);
```
And then you can remove the `const` declaration.